packages/create-next-app/index.ts

```diff
@@ -83,6 +83,12 @@ const program = new Commander.Command(packageJson.name)
     `
 
   Initialize inside a \`src/\` directory.
+`
+  )
+  .option(
+    '--turbo',
+    `
```

Comment:
```suggestion
    `
    
```

Reply:
It looks like the others have a line break here.

Reply:
Great catch, thank you!!